scripts/bug_classifier.py

```diff
@@ -69,10 +69,8 @@ def classify_bugs(model_name, classifier, bug_id):
         else:
             probas = model.classify(bug, probabilities=True, importances=False)
 
-        if np.argmax(probas) == 1:
-            print(f"Positive! {probas}")
-        else:
-            print(f"Negative! {probas}")
+        pred_class = model.le.inverse_transform([np.argmax(probas)])[0]
+        print(f"{pred_class} {probas}")
```

Comment:
For 1 and 0, we can still print Positive and Negative (or True and False), it's a bit more user-friendly.